assets/question_catalog/definition.json

```
@@ -4269,8 +4281,10 @@
                "highway": "elevator",
                "door:height": false
             },
+            "osm_element": ["Node", "ClosedWay"],
             "!parent": [
               {
+                "osm_element": ["OpenWay", "ClosedWay"],
```

I think it makes sense to only look for an `OpenWay` here.

```suggestion
                "osm_element": "OpenWay",
```